app/assets/stylesheets/components/_buttons.css

```diff
@@ -0,0 +1,19 @@
+.button-holder {
```

Comment:
**issue**: Naming is a bit inconsistent. `button-holder` vs `yellow-button`. Better would be `button-holder` vs `button-yellow`.

Normally I like to follow a few guidelines when doing css (feel free to challenge them):

1. One file per component (so `_yellow-button.css` and `_button-holder.css`
2. extensibility, I think we'd benefit from having a `button` class with basic rules and derive from that, maybe using BEM principles:
```css
.button {
  display: flex;
  justify-content: center;
  align-items: center;
}

.button--yellow {
    background-color: var(--yellow);
}
```

```html
<a class="button button--yellow"></a>
```

Reply:
agreed. sometimes I still find myself confused regarding how to name things 😬